DQM/SiStripMonitorTrack/interface/SiStripMonitorTrack.h

```diff
@@ -233,9 +233,12 @@ class SiStripMonitorTrack : public DQMEDAnalyzer {
   edm::ParameterSet Parameters;
   edm::InputTag Cluster_src_;
 
+  edm::EDGetTokenT<edm::DetSetVector<SiStripDigi> > digiToken_;
   edm::EDGetTokenT<edmNew::DetSetVector<SiStripCluster> > clusterToken_;
   edm::EDGetTokenT<reco::TrackCollection> trackToken_;
 
+  edm::Handle<edm::DetSetVector<SiStripDigi>> digiHandle_;
```

Comment:
But that's part of the point...
Since it is a class member it makes it incredibly easy to misuse (ie outside analyze), 
the fact you have a very complex call tree makes it even easier.